docs/index.mdx

```diff
@@ -22,3 +22,11 @@ Here you’ll find everything you need to start working with [Saleor](https://sa
 [![](icons/getting-started.svg) **Developer's Guide** Build and deploy a store based on Saleor.](developer.mdx)
 
 </CardGrid>
+
+## Api Reference
+
+<CardGrid home>
+
+[![](icons/getting-started.svg) **Api Reference** Detailed description of the whole schema.](api-reference/api-introduction.mdx)
```

Comment:
```suggestion
[![](icons/getting-started.svg) **API Reference** A detailed description of the entire GraphQL API.](api-reference/api-introduction.mdx)
```

Also, there is a better icon that was removed when the previous reference was dropped: https://github.com/mirumee/saleor-docs/commit/962b221e4ca5935ee6a5fbe511f35e18f7774062